ReactAndroid/src/main/java/com/facebook/react/views/checkbox/ReactCheckBoxManager.java

```diff
@@ -60,4 +65,76 @@ public void setOn(ReactCheckBox view, boolean on) {
     view.setOn(on);
     view.setOnCheckedChangeListener(ON_CHECKED_CHANGE_LISTENER);
   }
+
+  @ReactProp(name = "checkedColor", customType = "Color")
+  public void setCheckedTintColor(ReactCheckBox view, Integer color) {
+    String defaultColorIdOfUncheckedState = "colorPrimaryDark";
+    String defaultColorIdOfCheckedState = "colorAccent";
+    int stateBeingChanged = android.R.attr.state_checked;
+
+    setCheckboxTintList(view, color, stateBeingChanged, defaultColorIdOfUncheckedState, defaultColorIdOfCheckedState);
+  }
+
+  private void setCheckboxTintList(ReactCheckBox view, Integer colorToSet, int stateBeingChanged, String defaultColorIdOfPreservedState, String defaultColorIdOfChangedState) {
+    int defaultColorOfPreservedState = getThemeColor(view.getContext(), defaultColorIdOfPreservedState);
+
+    if (colorToSet == null) {
+      int defaultColorOfChangedState = getThemeColor(view.getContext(), defaultColorIdOfChangedState);
+      ColorStateList csl = createColorStateList(view, stateBeingChanged, defaultColorOfChangedState, defaultColorOfPreservedState);
+      CompoundButtonCompat.setButtonTintList(view, csl);
+    } else {
+      ColorStateList csl = createColorStateList(view, stateBeingChanged, colorToSet, defaultColorOfPreservedState);
+      CompoundButtonCompat.setButtonTintList(view, csl);
```

Comment:
would be nice to have only one `CompoundButtonCompat.setButtonTintList` call instead of two, but I'm afraid it would break the tests, so I'm leaving it as-is for now.